CHANGELOG.md

```diff
@@ -4,9 +4,10 @@
 
 ### Added
 - Added a Small Screen Mode, when you rezise the window to be small it will only show the chatlist with account sidebar or the Chat View with a back button.
-- show VCard attachement as VCard in message list #3840
+- show VCard attachment as VCard in message list #3840
 - add contact from VCard & start chat on click #3840
 - Webxdc realtime support #3741
+- send contact as VCard from attachment context menu #3830
```

Comment:
branch needs to be rebased to current `main` before merging and changelog entry needs to be moved up to current unreleased section